mtags/src/main/scala-2/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -212,6 +213,18 @@ case class ScalaPresentationCompiler(
       params.token
     ) { pc => new SignatureHelpProvider(pc.compiler()).signatureHelp(params) }
 
+  override def prepareRename(
+      params: OffsetParams
+  ): CompletableFuture[ju.Optional[Range]] =
+    compilerAccess.withNonInterruptableCompiler(
+      Optional.empty[Range](),
+      params.token
+    ) { pc =>
+      Optional.ofNullable(
+        new PcRenameProvider(pc.compiler(), params, None).prepareRename().orNull
+      )
```

Comment:
```suggestion
       new PcRenameProvider(pc.compiler(), params, None).prepareRename().asJava
```
we should have an extension method for that somewhere